src/server/api/controllers/favorite.vessels.controller.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
```

Comment:
Please rename the file and update references to `favorite-vessels.controller.js`

Reply:
Done